README.adoc

```diff
@@ -131,20 +127,47 @@ InventoryLivenessCheck.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java[]
 ----
 
+A health Check for liveness allows third party services to determine if the application is running. 
```

Comment:
A health Check for liveness allows third party services to determine if the application is running. 
--->
A health check for liveness allows third party services to determine whether the application is running.

Reply:
I dont think this was resolved, otherwise LGTM